ktor-server/ktor-server-tests/jvm/test/io/ktor/tests/server/routing/RoutingResolveTest.kt

```diff
@@ -1041,6 +1041,125 @@ class RoutingResolveTest {
         }
     }
 
+    @Test
+    fun testRoutingWithWildcardTrailingPathParameter() = withTestApplication {
+        application.routing {
+            get("test/*") {
```

Comment:
I'm not sure that `*` is a mandatory parameter. What is a current behavior?

Reply:
from the doc
> * (wildcard) matches any path segment and can't be missing.